lambdas/questions/build.gradle

```diff
@@ -18,8 +18,6 @@ dependencies {
 			'org.apache.cxf:cxf-rt-frontend-jaxws:3.4.5',
 			'org.apache.cxf:cxf-rt-transports-http:3.4.5',
 			'javax.annotation:javax.annotation-api:1.3.2',
-			'org.apache.logging.log4j:log4j-api:2.17.1',
```

Comment:
As per:
https://docs.aws.amazon.com/lambda/latest/dg/java-logging.html#java-logging-log4j2
I believe these `log4j` dependencies are needed for LoggerFactory.getLogger to work

Reply:
Was trying to mininmise dependencies here, java lambda has a context-logger. There a multiple `build.gradle` files I only remove it in the lambda `build.gradle` in favor of the lambda logger.

Reply:
Reinstated